### PR TITLE
Finished pgSQL Build, Dummy Data

### PR DESCRIPTION
**Pull request by Cyberskout99**
Contains SQL script written for postgreSQL.  Zipped folder contains numerous CSV files that can be read into the Db.  We'll need to update the COPY statements to point at whatever directory the CSV files live in, once all the files are uploaded.

Please note all tables are written to delete/recompile with each re-start of the Db.  We will want to remove this functionality once the app is closer to production.  Foreign keys will need to be revisited as well.

### CODE REVIEW

**Review by aaronvbrown — CHANGES_REQUESTED** [review]
We'll need to replace FROM 'C:\Users\Paul\Desktop\SQL Files\' with references for this to work on anyone's machine but Paul's.  Please correct me if I'm wrong!

File: PDXwildlifeDb.sql [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+	InfName	varchar(25),
+	NumBirth	INTEGER,
+	PandaID_fk	INTEGER
+	--FOREIGN KEY(LitterID_fk) REFERENCES Litter_tbl(LitterID),
+	--FOREIGN KEY(PandaID_fk) REFERENCES Panda_tbl(PandaID),
+);
+-- Copying in from CSV template
+-- The Copy function will write in the PK (serial) variable automatically
+COPY Inf_tbl(InfID,LitterID_fk,NumBirth,InfDOB,PandaID_fk,InfName)
+FROM 'C:\Users\Paul\Desktop\SQL Files\Inf_tbl.csv' DELIMITER ',' CSV HEADER;
```
Comment (gittyeric) [inline PDXwildlifeDb.sql]: As Aaron mentioned, we can refactor this a bit to work on everyone's machines.  You'll need to add the csv files to the project itself (in some common-sense folder off the root, let's call it X).  Then you can use the 'relative' directory of './' to reference those csv files.  For example, C:\Users\Paul\Desktop\Inf_tbl.csv becomes './X/Inf_tbl.csv'.  As long as you then run the sql from the root of the project, everything should work.

**Comment by Cyberskout99** [comment]
Okay - I've edited the syntax in the SQL file I submitted before, I went with the example of 'X' directory.  Take a look now, and please let me know if any follow-up is required.

Thanks,
Paul

**Review by aaronvbrown — APPROVED** [review]
Let's try it out.  Thanks, Paul!

**Pull request merged**